### ISSUE TITLE
Button with IBAnimatable in iOS 14 title not display complete

### ISSUE THREAD

**Issue by MinaHardTask** [IBAnimatable/IBAnimatable#635]
when build project with Xcode 11 and run on iOS 14 button title display with ... 
![WhatsApp Image 2020-09-29 at 10 05 03 AM](https://user-images.githubusercontent.com/60383413/94541481-edc55c00-0247-11eb-99f1-30681e34be3c.jpeg)

**Comment by JakeLin** [comment]
@MinaHardTask are you using `AnimatableButton`? Is it an iOS 14 only issue?

**Comment by apphardtask** [comment]
> @MinaHardTask are you using `AnimatableButton`? Is it an iOS 14 only issue?

Yes, it's with iOS 14 only. But run the app using Xcode 11 (iOS 13), to get issue.

If we run with Xcode 12 (iOS 14), it's fine.

To reproduce this error, run the app with Xcode 11 & iPhone with iOS 14.
If you run the app with Xcode 12 & iPhone with iOS 14, it works fine.

**Comment by JakeLin** [comment]
> To reproduce this error, run the app with Xcode 11 & iPhone with iOS 14.
> If you run the app with Xcode 12 & iPhone with iOS 14, it works fine.

That's an interesting issue. So it sounds like the walkaround is to rebuild the app with Xcode 12, and everything works fine

**Comment by apphardtask** [comment]
> > To reproduce this error, run the app with Xcode 11 & iPhone with iOS 14.
> > If you run the app with Xcode 12 & iPhone with iOS 14, it works fine.
> 
> That's an interesting issue. So it sounds like the walkaround is to rebuild the app with Xcode 12, and everything works fine

@JakeLin 🤣 🤣 🤣

**Comment by JakeLin** [comment]
@apphardtask sometimes, it's difficult to fix some issue from Xcode or iOS itself. If possible, please use the walkaround to get around it first 🤣
